### ISSUE TITLE
onaddfile callback param order wrong

### ISSUE THREAD

**Issue by jstayton** [pqina/filepond#233]
Hi @rikschennink 👋 

### Summary

The [docs](https://pqina.nl/filepond/docs/patterns/api/filepond-instance/#methods) state that `onaddfile` passes `(error, file)` to the callback function, but I'm seeing `(file, error)`.

### Expected behaviour

I'd expect `(error, file)`, as that would match `onprocessfile` (which _does_ pass them in that order).

### Additional information

This is with v4.1.0 and Vue adapter v5.0.0.

**Comment by rikschennink** [comment]
Hi!

Maybe I'm missing something but:

```vue
<file-pond v-on:addfile="handleFilePondAddFile"/>
```

```js
methods: {
    handleFilePondAddFile: function(err, file) {
        console.log(err, file);
    }
}
```

In my test environment results in `null, file`

**Comment by jstayton** [comment]
Thanks for the quick response!

Hmmm, maybe it's related to file size and type validation plugins? I'm using both of those, and when either the file size is too large or the type is invalid, I get `file, error`.

I agree with what you're seeing though: when there isn't an error, I get `null, file`.

**Comment by rikschennink** [comment]
Okay, I'll take another look tomorrow.

**Comment by rikschennink** [comment]
Confirmed and fixed in version `4.1.2`, please confirm.

**Comment by jstayton** [comment]
Confirmed!

Thanks for the quick response and resolution!
